### PR TITLE
mpr121 implementation yaml docs

### PR DESCRIPTION
**Pull request by mvturnho**
## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphomeyaml](https://github.com/OttoWinter/esphomeyaml) with YAML changes (if applicable):** OttoWinter/esphomeyaml#<esphomeyaml PR number goes here>
**Pull request in [esphomelib](https://github.com/OttoWinter/esphomelib) with C++ framework changes (if applicable):** OttoWinter/esphomelib#<esphomelib PR number goes here>

## Checklist:
  - [ ] The documentation change has been tested and compiles correctly.
  - [ ] Branch: `next` is for changes and new documentation that will go public with the next esphomelib release. Fixes, changes and adjustments for the current release should be created against `current`.

### CODE REVIEW

File: components/sensor/mpr121.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+MPR121 Capacitive Touch Sensor
+===========================================
```
Comment (OttoWinter) [inline components/sensor/mpr121.rst]: Title bars must have same length as text

File: components/sensor/mpr121.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    :width: 20.0%
+
+    MPR121 Capacitive Touch Sensor.
+
+.. _Adafruit: https://learn.adafruit.com/adafruit-mpr121-12-key-capacitive-touch-sensor-breakout-tutorial/overview
+
+.. code-block:: yaml
+
+    # Example configuration entry
+	mpr121:
```
Comment (OttoWinter) [inline components/sensor/mpr121.rst]: This indentation appears to be faulty, mpr121: should have the same indentation as the comment above

File: components/sensor/mpr121.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	  id: mpr121_sensor
+	  address: 0x5A
+
+	binary_sensor:
```
Comment (OttoWinter) [inline components/sensor/mpr121.rst]: If this is a binary sensor you should create it in the binary_sensor folder, not sensor.

File: components/sensor/mpr121.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+	  - platform: mpr121
+		id: touch_key0
+		channel: 0
+		name: "touch_key0"
```
Comment (OttoWinter) [inline components/sensor/mpr121.rst]: Use a friendly name here, like "Touch Key 0"

File: components/sensor/mpr121.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+Configuration variables:
+------------------------
+
+The configuration is made up of twoe parts: The central component, and individual Binary sensors per channel.
```
Comment (OttoWinter) [inline components/sensor/mpr121.rst]:
```suggestion
The configuration is made up of two parts: The central component, and individual Binary sensors per channel.
```

**Review by OttoWinter — COMMENTED** [review]
- Please create this PR against the `next` branch, current is for docs fixes. next is for new features.

- Please do not duplicate the image files, `images/` only needs to contain one file called `mpr121.jpg` (also use script shown in contributing to scale it down, 105kb is way too much for main index)
-  Please convert tabs to spaces
- You need to add this file to `index.rst` for it to show up in the main table (see imgtabl in contributing guide)

**Comment by mvturnho** [comment]
close to make new PR for next branch

**Pull request closed without merging**